### ISSUE TITLE
how to set deserialize exclude 'class' field?

### ISSUE THREAD

**Issue by panyox** [dromara/soul#1232]
## Question

i don't need the 'class' field to return.

<img width="640" alt="WX20210407-141555@2x" src="https://user-images.githubusercontent.com/77569707/113819725-4c960000-97ac-11eb-9d79-cb69c3bd3250.png">

here is the class User

<img width="263" alt="WX20210407-141649@2x" src="https://user-images.githubusercontent.com/77569707/113819916-94b52280-97ac-11eb-9f4b-37c6c5f3da29.png">

any help? thanks

**Comment by dengliming** [comment]
Which plugin are you using?

**Comment by panyox** [comment]
> Which plugin are you using?

apache dubbo

**Comment by dengliming** [comment]
Can you provide your Interface definition?

**Comment by panyox** [comment]
> Can you provide your Interface definition? I'll try to test it locally.
```
public interface UserService {

    User getUser(Integer id);

    UserList getAllUsers();

}
```
```
@Service("userService")
public class UserServiceImpl implements UserService {

    @Override
    @SoulDubboClient(path = "/user")
    public User getUser(Integer id) {
        User user = new User();
        user.setId(id);
        user.setName("Jack");
        return user;
    }

    @Override
    @SoulDubboClient(path = "/all-users")
    public UserList getAllUsers() {
        UserList userList = new UserList();
        userList.setTotal(2);
        List<User> users = new ArrayList<>();
        User u1 = new User();
        u1.setId(1);
        u1.setName("张三丰");
        users.add(u1);
        User u2 = new User();
        u2.setId(2);
        u2.setName("周芷若");
        users.add(u2);
        userList.setUsers(users);
        return userList;
    }
}
```

```
public class User {
    private Integer id;
    private String name;
}
```
```
public class UserList {
    private Integer total;
    private List<User> users;
}
```

**Comment by panyox** [comment]
ok, thanks

**Comment by dengliming** [comment]
@panyox We fixed it at `master`. Can you try it again?

**Comment by panyox** [comment]
> @panyox We fixed it at `master`. Can you try it again?
yes, it works!
if it possible to make a patch release to maven, or i'll use the source code packaging it myself.
thank you

**Comment by dengliming** [comment]
We just released 2.3.0 in the last two days. Probably not so soon. You can keep an eye on it : )

**Comment by dengliming** [comment]
Closing. You can always reopen it if there are any problems
